docs/references/nextjs/route-protection-requiring-authentication.mdx

```diff
@@ -153,18 +145,21 @@ export async function GET() {
 ```
 
 ```tsx filename="app/api/dashboard/route.ts"
-import { auth } from "@clerk/nextjs/server";
+import { auth } from '@clerk/nextjs/server';
 
 export async function GET() {
   const { userId } = auth();
 
   if (!userId) {
-    return new Response('Error: No signed in user', { status: 401 })
+    return Response.json(
+      { error: 'Error: No signed in user' },
+      { status: 401 },
+    );
   }
 
   // Add your Route Handler logic here
 
-  return Response.json({ userId })
+  return Response.json({ userId: auth().userId });
```

Comment:
is this not redundant ? I thought the userId being passed is the auth().userId ?

Reply:
```suggestion
  return Response.json({ userId });
```